src/maxmind.cpp

```diff
@@ -188,6 +196,10 @@ List maxmind_bindings::lookup(CharacterVector ip_addresses, MMDB_s *mmdb_set,
       output.push_back(organization(mmdb_set, ip_addresses));
     } else if(fields[i] == "isp"){
       output.push_back(isp(mmdb_set, ip_addresses));
+    } else if(fields[i] == "autonomous_system_number"){
```

Comment:
might change the externally-facing bit of this to like asn/aso respectively just for user simplicity